conn_go18.go

```diff
@@ -126,3 +127,63 @@ func (cn *conn) cancel() error {
 		return err
 	}
 }
+
+var _converter = &converter{}
```

Comment:
Golint will probably error here. Why not call it `defaultConverter`? This would be clearer.

Reply:
Yes, you are right, I changed this variable name to defaultConverter